metadata.json

```diff
@@ -8,6 +8,10 @@
   "project_page": "https://github.com/djjudas21/puppet-freeradius",
   "issues_url": "https://github.com/djjudas21/puppet-freeradius/issues",
   "dependencies": [
+    {
+      "name": "puppet/systemd",
+      "version_requirement": ">=3.0.0 <4.0.0"
```

Comment:
I think we need to be `>=1.1.1` in order to get `systemd::dropin_file` with the params we're using (`filename` which I've suggested above is introduced in `1.1.1`, `systemd::dropin_file` without it is in `1.0.0`). I don't know if we want to test all the way back there, but I think if we can support older we should, as some users will be on older versions of `puppet/systemd`

Reply:
Also note that `camptocamp/systemd` is currently referenced here and in `.fixtures.yml`, so we would need to remove/update those references and ensure this is compatible (from memory, `camptocamp/systemd` became `puppet/systemd`?)